app/wama.core/ViewModel/User/UserAccountViewModel.cs

```diff
@@ -6,7 +6,7 @@ namespace WAMA.Core.ViewModel.User
     public abstract class UserAccountViewModel
     {
         public virtual UserAccountType AccountType { get; }
-
+        [Required]
```

Comment:
I meant in the UserAccountAddNewUser.cshtml . When you add an new patron account  and click add new user without filling the fields,it allows you to do that which it should not